daml-lf/engine/src/test/daml/BasicTests.daml

```diff
@@ -0,0 +1,383 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+
+
+module BasicTests where
```

Comment:
the engine tests only used the templates in this module, never the scenarios, so we remove them in this copy of the file